client/a8c-for-agencies/style.scss

```diff
@@ -210,7 +212,8 @@
 	}
 
 	.split-button .button {
-		font-size: 0.875rem;
+		//font-size: 0.875rem;
```

Comment:
Can we remove this comment?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/91798/commits/233eec3e4c20aebc5851415aec79cf22fcdbc6c0